### PR TITLE
Align ACS tasks output and annotations according to RHDH requirements

### PR DESCRIPTION
**Pull request by mmorhun**
image

### CODE REVIEW

**Review by jduimovich — APPROVED** [review]

**Review by jduimovich — DISMISSED** [review]
this is wrong -> `roxctl-deploy-check` it should be `roxctl-deployment-check`

same change in the oc annotation command in oc annotate.

**Comment by mmorhun** [comment]
Fixed

File: task/acs-deploy-check/0.1/acs-deploy-check.yaml [hunk 1]
```diff
@@ -39,7 +38,17 @@ spec:
...
-    - name: rox-deploy-scan
+    - name: annotate-task
+      image: registry.redhat.io/openshift4/ose-cli:4.13@sha256:73df37794ffff7de1101016c23dc623e4990810390ebdabcbbfa065214352c7c
+      script: |
+        #!/usr/bin/env bash
+        echo "acs-image-scan $(context.taskRun.name)"
+        oc annotate taskrun $(context.taskRun.name) task.results.format=application/json
+        oc annotate taskrun $(context.taskRun.name) task.results.type=roxctl-deployment-check
+        oc annotate taskrun $(context.taskRun.name) task.results.container=step-report
+        oc annotate taskrun $(context.taskRun.name) task.output.location=logs
```
Comment (chmeliik) [inline task/acs-deploy-check/0.1/acs-deploy-check.yaml:49]: This is needed because the annotations from the Task don't always get applied to the TaskRun?

**Review by chmeliik — APPROVED** [review]

File: task/acs-deploy-check/0.1/acs-deploy-check.yaml (hunk 1 above)
Comment (mmorhun) [reply task/acs-deploy-check/0.1/acs-deploy-check.yaml:49]: Yes, due to a bug in Tekton, they just removed.

**Review by mkosiarc — APPROVED** [review]

**Pull request merged**